subsys/fs/Kconfig.fatfs

```diff
@@ -88,6 +89,39 @@ config FS_FATFS_NUM_DIRS
 	  at compile-time.
 	  This affects use of fs_opendir on FAT type mounted file systems.
 
+config FS_FATFS_HAS_RTC
+	bool "Timestamping support"
+	help
+	  Enable file system timestamping instead of using a hardcoded date
+	  for all operations. Requires an application supplied implementation
+	  of `get_fattime`. Format of the uint32_t bits are as follows:
```

Comment:
There should be at least a test or sample providing example of reference implementation in context of Zephyr for the enabled function.